presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -1062,6 +1062,12 @@ standard rules for nulls. The subquery must produce exactly one column::
     SELECT name
     FROM nation
     WHERE regionkey IN (SELECT regionkey FROM region)
+ 
+``IN`` can also be used to test a column against a static list::
+ 
+    SELECT name
+    FROM nation
+    WHERE nationkey IN (3, 6, 9)
```

Comment:
Change this query to use regionkey instead so it makes more sense. Also for comparison add maybe both equivalent queries.

select name from nation where regionkey in (select regionkey from region where name='AMERICA' or name='AFRICA');

is the same as 

select name from nation where regionkey in (1,3);